tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -444,6 +444,8 @@ Confirm data replacement
 Delete second subject file
     [Tags]  HappyPath
     user clicks link  Back  # back from data replacement confirmation
+    user clicks link  Back  # second back from data replacement confirmation
```

Comment:
As an alternative to this I believe you can use `user reloads page`

Reply:
Ahh OK that's a good one to know, thanks!